### ISSUE TITLE
Linux ubuntu deb License

### ISSUE THREAD

**Issue by jkudish** [electron-userland/electron-builder#242]
When the produced deb is opened in the Ubuntu Software Center, there's a few details missing:

* Developer Website points to `example.com`
* License is missing
* No icon is shown

Is it possible to fill in those details? Pretty low priority, but would be nice to have this info.

**Comment by develar** [comment]
> No icon is shown

Even after #239? Please specify exactly where icon is missed.

> License is missing
 
Where it is missed?

> Developer Website points to example.com

Yes, we don't specify `--url` (https://github.com/jordansissel/fpm/wiki) "Add a url for this package. (default: "http://example.com/no-uri-given")"
I will add `build.linux.packageUrl`. Do you like this solution?

**Comment by jkudish** [comment]
These are missed in the Ubuntu Software Center that is opened when the deb is opened on an Ubuntu machine. (This is not related to #239, that one is fixed, thanks!). I am not even positive that it's possible to set an icon there, but there is a white empty box indicating a space for a possible "screenshot". Other apps (not electron) have that set. That's also where the license shows as "unknown".

> I will add build.linux.packageUrl. Do you like this solution?

Yeah that would be fine.

**Comment by develar** [comment]
`package url` fixed.
 
License is fixed, but not for Ubuntu. Will be addressed later.
